Documentation/getting_started/3_adding_content.md

```diff
@@ -0,0 +1,77 @@
+This file will go over adding content to your game. For help on creating a project please look at [Creating a New Project](getting_started/1_creating_a_new_project.md)
+
+First of all you are gonna need some content for your game. For this tutorial use the following image of a ball:
+
+![Open Content](images/getting_started/ball.png)
+
+Do **right-click > Save Image As** and save it somewhere with the name "ball.png".
+
+Now open up your game project and look at the left. You should see a solution explorer window. Expand the **Content** folder and open up **Content.mgcb** file by double clicking on it.
+
+![Open Content](images/getting_started/3_open_content.png)
+
+You should now see a MonoGame Pipeline Tool window open up. In case it didn't get opened, you can right-click on **Content.mgcb**, select **open with** and then select **MonoGame Pipeline**.
+
+![MonoGame Pipeline Tool](images/getting_started/3_pipeline_tool.png)
+
+Your game content is managed from this external tool. You can add content to your game in one of the following ways:
+ - **Add Existing Item** toolbar button
+ - **Edit > Add > Existing Item...** menu button
+ - **right-click > Add > Existing Item...** context menu
+ 
+In our case lets user the **Add Existing Item** toolbar button.
+
+![Add Content](images/getting_started/3_add_content.png)
+
+You should now be prompted to select a file. Select the "ball.png" that you have downloaded a moment ago. After that you will be asked on what action you want to do for adding the file. Just leave the it to default and click **OK**.
```

Comment:
@cra0zy we are removing loading of native assets at some point. I think we should be introducing people to the Content.mgcb file and how that all works rather than linking .png files. This was one of my biggest problems with the Xamarin documentation.

Reply:
I think you misread something, its showing how to add the .png file to Content.mgcb using Pipeline Tool.